pubsub-adapter/src/main/kotlin/com/ably/pubsub/RestPresence.kt

```diff
@@ -0,0 +1,39 @@
+package com.ably.pubsub
+
+import io.ably.lib.types.*
+
+interface RestPresence : Presence {
+
+  /**
+   * Retrieves the current members present on the channel and the metadata for each member,
+   * such as their [io.ably.lib.types.PresenceMessage.Action] and ID. Returns a [PaginatedResult] object,
+   * containing an array of [PresenceMessage] objects.
+   *
+   * Spec: RSPa
+   *
+   *  @param params the request params:
+   *  - limit (RSP3a) - An upper limit on the number of messages returned. The default is 100, and the maximum is 1000.
+   *  - clientId (RSP3a2) - Filters the list of returned presence members by a specific client using its ID.
+   *  - connectionId (RSP3a3) - Filters the list of returned presence members by a specific connection using its ID.
+   * @return A [PaginatedResult] object containing an array of [PresenceMessage] objects.
+   */
+  fun get(params: List<Param>): PaginatedResult<PresenceMessage>
```

Comment:
This is inconsistent with `RealtimePresence` in this PR, where we name the params explicitly. We should pick one approach for the wrapper.

Looks like in Java we use an array of params across the board, though admittedly I prefer the named arguments approach.

Reply:
Missed this, updated